### PR TITLE
feat(sumologicschemaprocessor): add docker stats metrics mapping

### PR DESCRIPTION
**Pull request by aboguszewski-sumo**

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
Could we add mapping to sumologicschema as part of this PR, or write some config to do this for customers? Should be pretty easy task and customers won't have to migrate their dashboards right now

**Comment by aboguszewski-sumo** [comment]
> Could we add mapping to sumologicschema as part of this PR

We could, but is it worth it? I mean, for other sources there would be probably a similar situation, so do we want to add all that there?

**Comment by sumo-drosiek** [comment]
> We could, but is it worth it? I mean, for other sources there would be probably a similar situation, so do we want to add all that there?

We should ensure that switching from Installed Collector to OTC will be as seamless as possible

**Comment by aboguszewski-sumo** [comment]
@sumo-drosiek I added the subprocessor, please review.

File: pkg/processor/sumologicschemaprocessor/config.go [hunk 1]
```diff
@@ -36,6 +37,7 @@ const (
 	defaultAddCloudNamespace           = true
 	defaultTranslateAttributes         = true
 	defaultTranslateTelegrafAttributes = true
+	defaultTranlateDockerMetrics       = true
```
Comment (aboguszewski-sumo) [inline pkg/processor/sumologicschemaprocessor/config.go:40]: This causes this change to be a breaking one, how careful are we going to be with that?

Comment (sumo-drosiek) [reply pkg/processor/sumologicschemaprocessor/config.go:40]: Lets make it optional and discuss next week

Comment (sumo-drosiek) [reply pkg/processor/sumologicschemaprocessor/config.go:40]: IMHO, it should be enable by default and it is rather lower severity

**Review by sumo-drosiek — APPROVED** [review]
Awesome!!!

File: pkg/processor/sumologicschemaprocessor/config.go (hunk 1 above)
Comment (aboguszewski-sumo) [reply pkg/processor/sumologicschemaprocessor/config.go]: I set it to "false". I'll add this topic to Monday meeting's agenda

**Pull request merged**